app/javascript/react/components/RestaurantShow.js

```diff
@@ -23,11 +25,34 @@ const RestaurantShow = (props) => {
     .catch(error => console.error(`Error in fetch: ${error.message}`))
     }, [])
 
-  return (
-      <h1>
-        {restaurant.name}
-      </h1>
-  )
+    const reviewTileArray  = () => restaurant.reviews.map((review) => {
```

Comment:
`reviewTileArray` does not need to be an arrow function, it can be defined as:
```
const reviewTileArray = restaurant.reviews.map( ...
```

Reply:
it needs to be a function because we only want it to execute if the condition in line 52(also below) is true 

```Object.keys(restaurant).length !== 0```